sigar_windows.go

```diff
@@ -82,7 +74,15 @@ func (self *ProcFDUsage) Get(pid int) error {
 func (self *Uptime) Get() error {
```

Comment:
Since we can't guarantee how the library will be used I would feel more comfortable if it was concurrency-safe. I will make a change.